src/main/java/htsjdk/samtools/util/GZIIndex.java

```diff
@@ -217,6 +217,19 @@ public String toString() {
         return "GZIIndex:" + StringUtil.join(", ", entries);
     }
 
+    /**
+     * Writes this index into the requested path.
+     *
+     * NOTE: This method will close out the provided output stream when it finishes writing the index
```

Comment:
@jamesemery This comment is misplaced here- this method has no `provided output stream`.